package chapter17;

import java.io.IOException;

public class Exercise4 {
    public static void main(String[] args) throws IOException {
        System.out.println("プログラムが起動しました");
        throw new IOException();
    }
}
